### PR TITLE
Updated strings and added German translation

### PR DESCRIPTION
**Pull request by dodekaphilist**
I reformatted the English strings and added inline links. I hope I chose the correct formatting option and the changes are welcome.

I also added the German translation.

### CODE REVIEW

File: values/strings.xml [hunk 1]
```diff
@@ -4,17 +4,17 @@
 <string name="language_set">Language set to {lang}</string>
 
 <string name="message_welcome">
-This bot transcribes audio and pictures into text.
-Add me to a group or forward audio messages and pictures to me!
+This bot transcribes audio and pictures into text. Add it to a group or forward audio messages and pictures to it.
+{b}Note{/b}: Within groups, the bot will respond to {b}admin commands only{/b}
```
Comment (turicas) [inline values/strings.xml]:
```suggestion
{b}Note{/b}: Within groups, the bot will respond to {b}admin commands only{/b} (make sure you're not an anonymous admin also).
```

File: values/strings.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@ Money from donations will be used for paying the server and backend costs and le
 
 <![CDATA[https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=9H5XK6UDCAS86]]>
 
-You can also donate with BTC, ETH or LTC
+You can also donate with Bitcoin, Ethereum oder Litecoin
```
Comment (turicas) [inline values/strings.xml]:
```suggestion
You can also donate with Bitcoin, Ethereum or Litecoin
```

Comment (dodekaphilist) [reply values/strings.xml]: Good catch!

**Comment by carloalbertobarbano** [comment]
Thanks and sorry the delay, I did not notice this PR earlier..

**Pull request merged**